images/tier1/sort-type.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
```

Comment:
Dave's right -- I think this on should be flipped in RTL by adding a `mirror-in-rtl="true"` attribute to the SVG. See this for more info:
https://github.com/BrightspaceUI/icons#bidirectionality